modules/electrophysiology_browser/jsx/react-series-data-viewer/src/eeglab/EEGLabSeriesProvider.tsx

```diff
@@ -106,20 +109,45 @@ class EEGLabSeriesProvider extends Component<CProps> {
           this.store.dispatch(setInterval(timeInterval));
         }
       }
-    ).then(() => Promise.race(racers(fetchText, epochsURL)).then((text) => {
+    ).then(() => Promise.race(racers(fetchText, epochsURL))
+      .then((text) => {
         if (!(typeof text.json === 'string'
           || text.json instanceof String)) return;
+        return tsvParse(
+          text.json.replace('trial_type', 'label'))
+          .map(({ onset, duration, label }, i) => ({
```

Comment:
npm isn't complaining that there's a space before onset and after label? Otherwise the code looks great.

Reply:
Eslint doesn't complain but I can fix it!

Reply:
I don't have a personal preference regarding the format of the code but I'm concerned why eslint isn't working? Today I encountered that before & after "space" issue for code I was writing for ccna.

Reply:
Maybe because it's a ts file and not a js file?

Reply:
Exactly, the ESLint config is not inclusing the .tsx at the moment:
```
   new ESLintPlugin({
        files: [
          'modules/',
          'jsx/',
          'jslib/',
          'htdocs/js/',
          'webpack.config.js',
        ],
        cache: true,
      }),
```

I can enable eslint a bit later once all the EEG PRs will be merged, since it will be some work.